Sources/SPMTestSupport/TestWorkspace.swift

```diff
@@ -38,7 +38,7 @@ public final class TestWorkspace {
         packages: [TestPackage],
         toolsVersion: ToolsVersion = ToolsVersion.currentToolsVersion,
         skipUpdate: Bool = false,
-        enablePubGrub: Bool = false
+        enablePubGrub: Bool = true
```

Comment:
Cool! I had a task for doing this, heh. Can you make this change in a separate PR?

Reply:
Done: https://github.com/apple/swift-package-manager/pull/2421